pkg/plugin/processor/builtin/impl/unwrap/debezium.go

```diff
@@ -64,14 +64,15 @@ func NewDebeziumProcessor(logger log.CtxLogger) sdk.Processor {
 func (d *debeziumProcessor) Specification() (sdk.Specification, error) {
 	return sdk.Specification{
 		Name:    "unwrap.debezium",
-		Summary: "Unwraps a Debezium record from the input OpenCDC record.",
+		Summary: "Unwraps a Debezium record from the input [OpenCDC](https://conduit.io/docs/features/opencdc-record).",
```

Comment:
Whoops, we just lost a record here, this is unacceptable for a data streaming tool! 🙈 
```suggestion
		Summary: "Unwraps a Debezium record from the input [OpenCDC record](https://conduit.io/docs/features/opencdc-record).",
```

Same goes for all other changes.